tests/testthat/test-add_n.tbl_uvregression.R

```diff
@@ -0,0 +1,70 @@
+skip_if_not(is_pkg_installed(c("broom.helpers", "broom"), reference_pkg = "gtsummary"))
+
+test_that("add_n.tbl_uvregression() works", {
+  expect_silent(
+    tbl1 <- tbl_uvregression(
+      trial,
+      x = trt,
+      include = c(marker, age),
+      show_single_row = trt,
+      method = lm,
+      hide_n = TRUE
+    )
+  )
+  # total N added to table is accurate
+  expect_error(
```

Comment:
We can use `expect_silent()` here to ensure there are no unwanted errors, warnings, or messages.